### DISCUSSION TITLE
No album cover when uploading songs to media

### DISCUSSION THREAD

**Discussion by fsan2val** [AzuraCast/AzuraCast#6675] [Q&A] [answered]
I am using azuracast on ubuntu 22.04, and on Arm64 vCPU. When uploading songs with a cover, it appears in the media section without the image, only the default or custom icon appears. I have tried many radios but it is the same. Another detail that I only use icecast. What can be?
![medios](https://github.com/AzuraCast/AzuraCast/assets/69484466/83472e7b-4317-46cf-b5da-dafc5ddef83a)

**Comment by BusterNeece** [comment, marked as answer]
This may have to do with what kind of ID3 tags you're using on your media. Be sure you're using ID3v2, not ID3v1, if your media tagging software can distinguish between the two.

**Reply by Moonbase59** [reply]
Yep, and try not to use "exotic" cover image file formats. Many libraries can’t correctly handle progressive JPEGs, WEBP, TIFF, etc. Best to stick with non-progressive JPEG and/or PNG in not to large sizes. I usually keep hi-res front and back covers in the album folder, and use a 500x500 JPEG _inside_ the audio file. Image captions inside the audio file can also be problematic, simply don’t use any.

**Reply by fsan2val** [reply]
Thank you very much for your recommendations. In fact, I tried and I was able to see the images on the azuracast platform with ID3v2

Thank you very much to all
